redshells/train/train_graph_convolutional_matrix_completion.py

```diff
@@ -59,7 +66,13 @@ def run(self):
         ratings = df[self.rating_column_name].values
 
         model = GraphConvolutionalMatrixCompletion(
-            user_ids=user_ids, item_ids=item_ids, ratings=ratings, item_features=item_features, **self.model_kwargs)
+            user_ids=user_ids,
+            item_ids=item_ids,
+            ratings=ratings,
+            user_features=user_features,
+            item_features=item_features,
+            test_size=0.2,  # TODO
```

Comment:
"todo" is remaining. I think it is OK to hand over to next improvement

Reply:
I use class member variable for `test_size`. Thanks!